ballerina/natives.bal

```diff
@@ -16,17 +16,17 @@
 
 import ballerina/jballerina.java;
 
-final StatisticConfig[] DEFAULT_GAUGE_STATS_CONFIG = [{ timeWindow: 600000, buckets: 5,
+isolated final StatisticConfig[] DEFAULT_GAUGE_STATS_CONFIG = [{ timeWindow: 600000, buckets: 5,
```

Comment:
Can't this be immutable instead?

Also naming convention is to use uppercase with constants only. (Not introduced by this change).

```suggestion
final StatisticConfig[] & readonly defaultGaugeStatsConfig = [{ timeWindow: 600000, buckets: 5,
```

If this is possible, we can get rid of clones and lock when accessing this.

Reply:
This is possible. I will address the requested changes